fabtests/benchmarks/dgram_pingpong.c

```diff
@@ -101,12 +101,19 @@ int main(int argc, char **argv)
 			ft_parseinfo(op, optarg, hints, &opts);
 			ft_parsecsopts(op, optarg, &opts);
 			break;
+		case 1:
+			if (ft_parse_pin_core_option(argv[optind-2],
+						long_opts[lopt_idx].name, optarg) < 0)
+				return EXIT_FAILURE;
+			break;
 		case '?':
 		case 'h':
 			ft_csusage(argv[0], "Ping pong client and server using UD.");
 			ft_benchmark_usage();
 			FT_PRINT_OPTS_USAGE("-T <timeout>",
 					"seconds before timeout on receive");
+			FT_PRINT_OPTS_USAGE("--pin_core <core number>",
+					"Specify the core to pin");
```

Comment:
We need some mechanism for handling the long options that doesn't require so much code duplication.  I'm not quite sure what that looks like.  Could we do something like:
```
getopt_long(argc, argv, ... LONG_OPTS ...)
...
default:
    if (!ft_parse_long_opts(...))
        continue;
    /* fall through */
case '?':
case 'h':
    fd_longopts_usage();
```
The long opts may not apply to every test, but it should be harmless to set them.  We have other opts that follow this model, where some tests ignore them.

Maybe this starts pushing us to have a more centralized function for opt processing.